### PR TITLE
Basket Data & MoneyBar #28

### PR DESCRIPTION
**Pull request by matthall8**
Updates:

- Basket gets state from Redux #80 
- User can remove item from basket #83 
- The user's quantity of a certain meal is present #84 
- Cost updates in the price bar #75

### CODE REVIEW

File: client/src/pages/BasketPage/BasketPage.js [hunk 1]
```diff
@@ -1,27 +1,79 @@
...
-          <OrderInfo>
-            <Cost>Cost: £5.00</Cost>
-            <select>
-              <option value="">1 Portion</option>
-            </select>
-            <button>Delete</button>
-          </OrderInfo>
-        </MealCard>
-        <MoneyButton />
+        {this.basketIsEmpty() ? (
```
Comment (DenisKent) [inline client/src/pages/BasketPage/BasketPage.js]: I think this deserves more than the if shorthand. It wilkl more readable if you did something like:
if(this.basketIsEmpty()) 
return(x) 
else 
return(y)

**Comment by Karyum** [comment]
ughhh why is PR 4 issues ?

**Comment by matthall8** [comment]
> ughhh why is PR 4 issues ?

Because it is part of an epic..

File: client/src/pages/BasketPage/BasketPage.js [hunk 2]
```diff
@@ -1,27 +1,76 @@
...
+import { Link } from "react-router-dom";
+import removeBasketItem from "../../actions/removeBasketItem";
+import { MealCard } from "../../components";
 import { MealName, Delivery, OrderInfo, Cost } from "./BasketPage.style";
 
-import mealImage from "../../assets/meal_image.png";
-
-export default class MealListPage extends Component {
+class BasketPage extends Component {
+  basketIsEmpty = () => Object.keys(this.props.basket).length === 0;
```
Comment (Karyum) [inline client/src/pages/BasketPage/BasketPage.js]: why does this need to be a function ?

File: client/src/pages/BasketPage/BasketPage.js [hunk 3]
```diff
@@ -1,27 +1,76 @@
...
-            <select>
-              <option value="">1 Portion</option>
-            </select>
-            <button>Delete</button>
-          </OrderInfo>
-        </MealCard>
-        <MoneyButton />
-      </React.Fragment>
-    );
+    if (this.basketIsEmpty()) {
```
Comment (Karyum) [inline client/src/pages/BasketPage/BasketPage.js]: :arrow_up:

File: client/src/pages/BasketPage/BasketPage.js (hunk 2 above)
Comment (matthall8) [reply client/src/pages/BasketPage/BasketPage.js]: Readability. I'll change.

**Pull request merged**